### ISSUE TITLE
[navigation bar] alter item ordering

### ISSUE THREAD

**Issue by diegoberaldin** [diegoberaldin/RaccoonForLemmy#1027]
Originally requested by @SagXD on Matrix:

> move setting to sidebar and move profile to far right

**Comment by diegoberaldin** [comment]
I do not agree partially on this and I think that, as long as we are a community, we should decide together. Any other opinions?

**Comment by SagXD** [comment]
> I do not agree partially on this and I think that, as long as we are a community, we should decide together. Any other opinions?

I don't know why I suggested this after thinking for a while I think current order is already good.

**Comment by diegoberaldin** [comment]
I would like to hear other opinions because, for example:
- it would be nice to rearrange order of items
- it would be nice to have different items (e.g. bookmarks instead of settings)
- rather than having things on the navigation drawer on the left, I can always move items on a side menu (on the right) and this would declutter the profile screen which currently has too many shortcuts on top.

**Comment by SagXD** [comment]
Ask in Lemmy Community for other opinion on this.

**Comment by N7-X** [comment]
Not sure I have an opinion about this one - it _did_ take me a while to get used to the settings location after I first installed the app. 
Part of this may be because I used to use Relay for Reddit, and they put theirs at the bottom of the side panel, so at that point I would have been in favor of the move for muscle memory reasons. 
I'm used to it now, and I don't think I really have a preference either way.
